Modules/_hashopenssl.c

```diff
@@ -111,6 +111,7 @@ newEVPobject(PyObject *name)
 
     retval->ctx = EVP_MD_CTX_new();
     if (retval->ctx == NULL) {
+        Py_DECREF(retval);
```

Comment:
Is it safe to call EVP_dealloc() here? Are all fields initialized to 0/NULL? (self->name)

(It's not a trap, I don't know the answer.)